modules/openxr/editor/openxr_action_map_editor.cpp

```diff
@@ -288,17 +290,17 @@ void OpenXRActionMapEditor::_load_action_map(const String p_path, bool p_create_
 	} else if (p_create_new_if_missing) {
 		action_map.instantiate();
 		action_map->create_default_action_sets();
-		action_map->set_path(p_path);
+		action_map->set_path(path);
```

Comment:
This line shouldn't be needed. `ResourceSaver` already sets that when saving.

Reply:
Ok, I've removed this line. Now that I've switched to the simpler approach you recommended, this change is a little out of place, but hopefully still OK :-)